things/api/things/http/endpoint_test.go

```diff
@@ -1972,23 +1972,23 @@ func TestCreateConnections(t *testing.T) {
 			body:        "{",
 		},
 		{
-			desc:        "Empty channelIDs",
+			desc:        "Connect valid things with empty channelIDs",
```

Comment:
Sorry for the detail but please use lowercase for the first letter. Maybe also `channel IDs`. 
There are also 2 descriptions in this test that can be more detailed. It would be nice if you could do it in this PR.

Reply:
I forgot Go prefers not to capitalize messages. Old habit. I'll change the ids so it doesn't read as matching the variable names.

I added connect to the other descriptions.